cmd/testscript/help.go

```diff
@@ -14,7 +14,7 @@ The testscript command runs github.com/rogpeppe/go-internal/testscript scripts
 in a fresh temporary work directory tree.
 
 Usage:
-    testscript [-v] [-e VAR]... files...
+    testscript [-v] [-e VAR]... [-update] files...
```

Comment:
Or `-u` ? :shrug:

Reply:
That works too. As ever, I'm not that fixed to any given name 👍